src/CMakeLists.txt

```diff
@@ -87,7 +104,17 @@ endforeach()
 # better than trying to fight the "rust way" here. It also includes doc tests,
 # which normally don't have persistent executables at all.
 add_test(NAME rust-unit-tests
-         COMMAND bash -c "${CARGO_ENV_VARS} cargo test ${RUST_TARGET_FLAG} ${RUST_BUILD_FLAG} --manifest-path \"${CMAKE_CURRENT_SOURCE_DIR}\"/Cargo.toml --target-dir \"${TARGETDIR}\" --features \"${RUST_FEATURES}\"")
+         COMMAND bash -c " \
+            ${CARGO_ENV_VARS} \
+            cargo test \
+            --workspace --exclude shadow-tests \
```

Comment:
Optional - Do we need to exclude shadow-tests here? If there aren't any unit tests there, it shouldn't do anything, but if we write unit tests e.g. for test utilities we'd probably want to run them here?

Reply:
Removed this line.